core/config/docs/docs_test.go

```diff
@@ -124,3 +124,23 @@ func assertTOML[T any](t *testing.T, fallback, docs T) {
 	fs, ds := string(fb), string(db)
 	assert.Equal(t, fs, ds, diff.Diff(fs, ds))
 }
+
+// Exclude TRON-specific custom config from the nil-check (required only for TRON Chain Type)
+// TODO: restructure with TRON specific config
+func assertFieldsNotNil_ExceptHTTPURLExtraWrite(t *testing.T, c chainlink.Config) {
```

Comment:
Ah sorry, looks like we have a lint rule here

```suggestion
func assertFieldsNotNilExceptHTTPURLExtraWrite(t *testing.T, c chainlink.Config) {
```